openshift-kube-apiserver/admission/network/externalipranger/externalip_admission_test.go

```diff
@@ -27,7 +27,7 @@ func fakeAuthorizer(t *testing.T) authorizer.Authorizer {
 	}
 }
 
-func (a *fakeTestAuthorizer) Authorize(attributes authorizer.Attributes) (authorizer.Decision, string, error) {
```

Comment:
can this commit merge into the one which introduces this code?

Reply:
Done. Squashed along with `UPSTREAM: <carry>:Add a RBAC checker for external IP ranger` into `UPSTREAM: <carry>: openshift-kube-apiserver: add kube-apiserver patches`